Dockerfile.prod

```diff
@@ -0,0 +1,19 @@
+# FROM golang:alpine
+# WORKDIR /app
+
+# COPY . .
+# RUN go mod download
+# # RUN CGO_ENABLED=0 GOOS=linux go build -o /doentre
+
+# EXPOSE 8080
+# CMD ["/doentre"]
```

Comment:
Do we need to keep this commented-out code?

Reply:
No problem. I'll remove it!